packages/server/test/support/fixtures/projects/non-proxied/cypress/integration/spec.js

```diff
@@ -0,0 +1,21 @@
+const serverUrl = `http://localhost:${Cypress.config('port')}`
+
+context('non proxied e2e', () => {
+  it('has the expected look', () => {
+    cy.task('get:tmp:path')
+    .then((path) => {
+      const pngPath = `${path}/cy-non-proxied.png`
+
+      // use google chrome to screenshot what the server shows
+      cy.exec(`google-chrome --headless --disable-gpu --no-sandbox --user-data-dir=${path}/nonproxiedprofile --screenshot=${pngPath} ${serverUrl}`)
```

Comment:
i think this will fail for non-linux OS's.

is the intention to only run this test in CI?

Reply:
does `google-chrome` not exist in the path on macOS? I thought this would work for macOS and Linux

Reply:
ok, this won't work on mac, but this will:

```js
cy.exec(`"${Cypress.browser.path}" --headless --disable-gpu --no-sandbox --user-data-dir=${path}/nonproxiedprofile --screenshot=${pngPath} ${serverUrl}`)
```

updated, and made it so the e2e test runs in chrome so this can work